frontend/libs/studio-components/src/components/StudioAvatar/StudioAvatar.module.css

```
@@ -0,0 +1,5 @@
+.avatar {
+  width: 40px;
+  height: 40px;
+  border-radius: 30px;
+}
```

We can define a `--avatar-size` variable scoped within the `.avatar` class to manage both width and height. This ensures consistency and makes it easier to adjust the avatar’s size. Additionally, we can use this variable to calculate the border-radius, ensuring the avatar stays perfectly round regardless of the size.

```suggestion
.avatar {
  --avatar-size: 40px;
  width: var(--avatar-size);
  height: var(--avatar-size);
  border-radius: calc(var(--avatar-size) / 2);
}
```